### PR TITLE
Added reflection on class so far

### PR DESCRIPTION
**Pull request by ghost**
This is a make-up assignment.

### CODE REVIEW

File: _posts/intjnotinfp/2020-12-02-reflection-post-intjnotinfp.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+author: intjnotinfp
+title: " intjnotinfp's Course Reflection Post
```
Comment (MishaCienfuegos) [inline _posts/intjnotinfp/2020-12-02-reflection-post-intjnotinfp.md]: Looks like you're missing a `"` at the end of the line here. Without it, the string never closes and your post won't show up correctly on the class site. Could you add that in?

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
Great use of markdown in your post! Just one small request in the header and this should be good to go. 🎰

**Review by ghost — COMMENTED** [review]
Added missing quotation

**Comment by ghost** [comment]
@MishaCienfuegos Could you review the heading change and merge?

**Review by MishaCienfuegos — APPROVED** [review]
Looks great, thanks!

**Pull request merged**